music21/expressions.py

```diff
@@ -1426,6 +1426,65 @@ def numberOfMarks(self, num):
             raise TremoloException('Number of marks must be a number from 0 to 8') from ve
 
 
+class ArpeggioException(exceptions21.Music21Exception):
+    pass
+
+
+class ArpeggioMark(Expression):
+    '''
+    ArpeggioMark must be applied to a Chord (not to a single Note).
+
+    The parameter arpeggioType can be 'normal' (a squiggly line), 'up' (a squiggly line
+    with an up arrow), 'down' (a squiggly line with a down arrow), or 'non-arpeggio' (a
+    bracket instead of a squiggly line, used to indicate a non-arpeggiated chord
+    intervening in a sequence of arpeggiated ones).
+
+    >>> am = expressions.ArpeggioMark('normal')
+    >>> am.type
+    'normal'
+
+    >>> am = expressions.ArpeggioMark('down')
+    >>> am.type
+    'down'
+    '''
+    def __init__(self, arpeggioType: str = 'normal'):
+        super().__init__()
+        if arpeggioType not in ('normal', 'up', 'down', 'non-arpeggio'):
+            raise ArpeggioException(
+                'Arpeggio type must be "normal", "up", "down", or "non-arpeggio"'
+            )
+        self.type = arpeggioType
+
+
+class ArpeggioMarkSpanner(spanner.Spanner):
+    '''
+    ArpeggioMarkSpanner is a multi-staff or multi-voice (i.e. multi-chord) arpeggio.
+    The spanner should contain all the simultaneous Chords that are to be arpeggiated
+    together.  It should not contain individual notes.
```

Comment:
Does this limitation come from musicxml? A three-staff StaffGroup with a single note on each staff could still yield an arpeggiated trichord? Or dyad on two staves? 🤔 

It's worth just a bit of thought to make sure we can make music21 as liberal as possible even if musicxml is more conservative.

Reply:
If there's implementation work to do, then we can totally punt that to a future PR. I'm just asking if there's any reason we have to reaffirm the musicxml limitation right out of the gate.

Reply:
I've removed this comment.  It's still true of ArpeggioMark, since an arpeggio on a single note is meaningless, but ArpeggioMarkSpanner could span a bunch of single notes easily.